src/app/admin/search-bangumi/result-detail/result-detail.html

```diff
@@ -7,35 +7,35 @@
         </a>
     </div>
     <div class="detail-container">
-        <form class="bangumi-form ui form" novalidate [formGroup]="bangumiForm">
+        <form class="bangumi-form ui form" novalidate>
             <div class="bangumi-image">
-                <img [src]="bangumi?.image" alt="image">
+                <img [src]="bangumi?.coverImage.url" alt="image">
             </div>
             <div class="bangumi-basic-info">
                 <div class="field">
                     <label>番组名</label>
-                    <input type="text" formControlName="name" name="name"/>
+                    <input type="text" [value]="bangumi?.name" readonly/>
                 </div>
                 <div class="field">
                     <label>中文名</label>
-                    <input type="text" formControlName="name_cn" name="name_cn"/>
+                    <input type="text" [value]="bangumi?.nameCn" readonly />
                 </div>
                 <div class="field">
                     <label>简介</label>
-                    <textarea type="text" formControlName="summary" name="summary" cols="20" rows="6"></textarea>
+                    <textarea type="text" [value]="bangumi?.summary" readonly cols="20" rows="6"></textarea>
                 </div>
                 <div class="field">
                     <label>放送开始</label>
-                    <input type="text" formControlName="air_date" name="air_date"/>
-                </div>
-                <div class="field">
-                    <label>放送星期</label>
-                    <input type="text" formControlName="air_weekday" name="air_weekday"/>
-                </div>
-                <div class="field">
-                    <label>话数</label>
-                    <input type="text" formControlName="eps" name="eps" readonly>
+                    <input type="text" [value]="bangumi?.airDate" readonly/>
                 </div>
+<!--                <div class="field">-->
```

Comment:
Why are those commented out?

Reply:
It's not supported by the new API. I will remove those lines.